sdk/python/requirements/py3.7-requirements.txt

```diff
@@ -0,0 +1,202 @@
+#
```

Comment:
Why are we adding this back?

Reply:
ah lol, I didnt realize we had already removed 3.7 requirements.